jvector-examples/src/main/java/io/github/jbellis/jvector/example/Bench.java

```diff
@@ -220,7 +240,14 @@ private static void gridSearch(DataSet ds, List<Integer> pqGrid, List<Integer> m
         try {
             for (int M : mGrid) {
                 for (int efC : efConstructionGrid) {
-                    testRecall(M, efC, pqGrid, diskOptions, efSearchFactor, ds, testDirectory);
+                    var graphPath = testDirectory.resolve("graph" + M + "-" + efC + ds.name);
+                    try {
+                        GraphResult graph = buildGraph(M, efC, ds, graphPath);
```

Comment:
is this actually an improvement?  we have to create a wrapper class in order to ... avoid building the graph close to where it's used?